docs/content/ActionList.mdx

```diff
@@ -196,29 +196,29 @@ const SelectFields = () => {
   }
 
   return (
-    <ActionList selectionVariant="multiple">
-      <ActionList.Group title="Visible fields">
+    <>
+      <ActionList selectionVariant="multiple" headingProps={{title: 'Visible fields'}}>
```

Comment:
Need to update this API. 

Also, I think we need to change this example's heading? Instead of `With Groups`, maybe `With Heading`?